packages/flutter/test/material/tab_bar_theme_test.dart

```diff
@@ -20,14 +20,22 @@ const List<Tab> _tabs = <Tab>[
   Tab(text: _tab3Text, icon: Icon(Icons.looks_3)),
 ];
 
-Widget _withTheme(TabBarTheme theme) {
+final List<SizedBox> _sizedTabs = <SizedBox>[
```

Comment:
I cannot use const here because the UniqueKeys are not constant values. Should I just leave them as `final`, or perhaps I can add text into them as with `_tabs` and find them that way?

Reply:
Good point. Making _sizedTabs final is fine. I thought looking things up via _sizedTabs[index].key was pretty slick.